src/get-started/install.md

```diff
@@ -44,7 +45,7 @@ use
 
 ```shell
 apt-get install build-essential libsodium-dev libsnappy-dev \
-    librocksdb-dev pkg-config
+    librocksdb-dev pkg-config libprotobuf-dev protobuf-compiler
```

Comment:
Had anyone check this command? I thought that these packages are outdated in Ubuntu (and I remember some reports from other developers about compilation issues with outdated protoc)

Reply:
works fine on Ubuntu 18.04.1, checked today

Reply:
not sure about 16.04 or others, maybe they need to update ppa repositories or install protoc manually

Reply:
> I thought that these packages are outdated in Ubuntu

On 16.04 they are, one needs to add a PPA (that we also use on Travis). Protobuf isn't the most recent there either, but it seems to work for me (but I haven't run _Exonum_ test suite with that, so can't tell for sure).

Reply:
Sorry for annoying, but will it be merged? Maybe I did something wrong or i have to do something more? Asking, because I want to understand whether to send PR in the future.